infra/modular-examples/aws-msft-365/main.tf

```diff
@@ -400,4 +400,30 @@ output "cognito_identities" {
 
 output "cognito_identity_pool_id" {
   value = module.cognito-identity-pool.pool_id
-}
\ No newline at end of file
+}
+
+//q: better to invert control, let calling modules write the files??
```

Comment:
I think so; so the caller will decide how to proceed with the outputs (by default, writing them into files)